dashboard/config/environments/development.rb

```diff
@@ -62,4 +62,7 @@
 
   # see stack traces around sql queries in the log
   # ActiveRecordQueryTrace.enabled = true
+
+  # act like a levelbuilder. Set to true if you want to be able to create levels or test levelbuilder functionality
+  config.levelbuilder_mode = false
```

Comment:
Or a config param that applies across the environments, `locals.yml` `levelbuilder_mode: true`, so toggling this on an e.g. staging or test server would be possible without changing code